src/main/webapp/WEB-INF/templates/user_dashboard.jsp

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>User Dashboard</title>
+    <script src="data/languages.js"></script>
+    <script src="js/user.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="/css/user_style.css">
+</head>
+<body>
+  <div class="main">
```

Comment:
This div doesn't really do anything that body doesn't. If you need to add font-family CSS or whatnot to make it global, add it to body.